packages/ui/src/components/va-progress-bar/_variables.scss

```
@@ -3,6 +3,7 @@
   --va-progress-bar-position: relative;
   --va-progress-bar-overflow: hidden;
   --va-progress-bar-height: 0.5rem;
+  --va-progress-bar-border-radius: 9999999px;
```

What if we do it like this? It's still rounded. 
```suggestion
  --va-progress-bar-border-radius: var(--va-progress-bar-height);
```